.actions/assistant.py

```diff
@@ -129,7 +145,7 @@ def _prune_packages(req_file: str, packages: Sequence[str]) -> None:
             req = list(pkg_resources.parse_requirements(ln_))[0]
             if req.name not in packages:
                 final.append(line)
-        pprint(final)
+        logging.info(final)
```

Comment:
You can just use print, it will appear if you do `pip ... -v`

Reply:
I don't think we really need it :)

Reply:
The installation process should not setup the `logging` module. A print is good enough